docs/puppeteer.md

```diff
@@ -0,0 +1,101 @@
+# Recipes Puppeteer with Lighthouse
+
+**Note**: https://github.com/GoogleChrome/lighthouse/issues/3837 tracks the discussion for making Lighthouse work in concert with Puppeteer.
+Some things are possible today (login to a page using Puppeteer, audit it using Lighthouse) while others (A/B testing the perf of UI changes) are not.
+
+### Custom network throttle settings using Puppeteer
```

Comment:
Could we advertise a different example of using puppeteer? Perhaps just logging in to your page with the second example you've already got? 

Our preferred method of customizing throttling is already [documented](https://github.com/GoogleChrome/lighthouse/blob/e8a1a7fb82c979f0298d8a7f752c61de467f7bcb/docs/throttling.md) and jumping in to muck with those settings in puppeteer is bound to result in over confidence in the accuracy of the performance metrics for those conditions.

Reply:
Yea that's fair. I can cook up a slightly different example of doing something in pptr then use lighthouse.